tools/deployment/package-helm/templates/topology-metrics-emitter-job.yaml

```
@@ -0,0 +1,47 @@
+            - "--fail"
+            - "--retry"
+            - "5"
+            - "--retry-all-errors"
+            - "--retry-delay"
+            - "1"
+            - "-X"
+            - "POST"
+            - "-H"
+            - "Content-Type: application/json"
+            - "-d"
+            - {{ include "clp.topologyMetricsPayload" . | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)